utils/common/src/main/java/org/apache/brooklyn/util/text/Strings.java

```diff
@@ -827,6 +828,33 @@ public static String ies(Iterator<?> x) {
         return result;
     }
 
+    /**
+     * Tries to convert v to a list of strings.
+     * <p>
+     * If v is <code>null</code> then the method returns an empty immutable list.
+     * If v is {@link Iterable} or an <code>Object[]</code> then toString is called on its elements.
+     * If v is a {@link String} then a singleton list containing v is returned.
+     * Otherwise the method throws an {@link IllegalArgumentException}.
+     */
+    public static List<String> toStringList(Object v) {
```

Comment:
Would be good to add unit tests for this (but we can do that in a separate PR, given that this PR has just moved it from one place to another).